api/PushEvent.json

```
@@ -151,6 +151,46 @@
             "deprecated": false
           }
         }
+      },
+      "notification": {
+        "__compat": {
+          "spec_url": "https://w3c.github.io/push-api/#dom-pushevent-notification",
+          "support": {
+            "chrome": {
+              "version_added": false,
+              "impl_url": "https://crbug.com/382298314"
+            },
+            "chrome_android": "mirror",
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false,
+              "impl_url": "https://bugzil.la/1935325"
+            },
+            "firefox_android": "mirror",
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "18.5"
```

Same here, actually:

```suggestion
              "version_added": "18.4"
```